server/src/test/java/io/crate/types/NumericTypeTest.java

```diff
@@ -21,7 +21,7 @@
 
 package io.crate.types;
 
-import static io.crate.testing.Asserts.assertThat;
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
```

Comment:
Maybe worth making this extend `DataTypeTestCase` as that will also exercise reference resolvers and translog round trips?

Reply:
Good call. Done